pkg/controllers/machine/termination_test.go

```diff
@@ -42,34 +65,373 @@ var _ = Describe("Termination", func() {
 					},
 				},
 			},
+			Status: v1alpha5.MachineStatus{
+				ProviderID: test.RandomProviderID(),
+				Capacity: v1.ResourceList{
+					v1.ResourceCPU:    resource.MustParse("10"),
+					v1.ResourceMemory: resource.MustParse("100Mi"),
+					v1.ResourcePods:   resource.MustParse("110"),
+				},
+				Allocatable: v1.ResourceList{
+					v1.ResourceCPU:    resource.MustParse("8"),
+					v1.ResourceMemory: resource.MustParse("80Mi"),
+					v1.ResourcePods:   resource.MustParse("110"),
+				},
+			},
+		})
+		node.Finalizers = []string{v1alpha5.TerminationFinalizer}
+	})
+	It("should cordon, drain, and delete the Machine on terminate", func() {
```

Comment:
Odd that we would say this test does drain if there's no actual pods to drain here.